### DISCUSSION TITLE
[Feature] Is it possible to support the application store to modify the default directory for installation?

### DISCUSSION THREAD

**Discussion by a602017206** [1Panel-dev/1Panel#7314] [Q&A]
### 1Panel Version

v1.10.21-lts

### Please describe your needs or suggestions for improvements

自己用的nas，如果默认安装在/opt下，会占用根目录的磁盘，我需要安装到我其它的盘里面，opt本身的空间太小了，nas是产品是绿联

**Comment by mjsoftking** [comment]
你把1panel安装到其他目录就好了

**Comment by a602017206** [comment]
我是用容器启动的，启动命令是下面的内容，/opt的映射已经改了，1p本身没问题，但是安装应用还是会到opt下，需要清了数据重新初始化1p吗？
docker run -d \
    --name 1panel \
    --restart always \
    --network host \
    -v /var/run/docker.sock:/var/run/docker.sock \
    -v /volume1/@docker/volumes:/var/lib/docker/volumes \
    -v /volume1/1Panel:/opt \
    -v /root:/root \
    -v /volume1:/volume1 \
    -v /volume2:/volume2 \
    -e TZ=Asia/Shanghai \
    moelin/1panel:latest

**Comment by wanghe-fit2cloud** [comment]
1Panel 目前尚未推出容器化安装方式或直接适配 NAS 的版本。如果在使用过程中遇到问题，建议在 discussion 中继续讨论。

**Comment by a602017206** [comment]
https://hub.docker.com/r/moelin/1panel   这个不是官方发布的？

**Reply by wanghe-fit2cloud** [reply]
这是社区用户维护的。
GitHub: https://github.com/okxlin/docker-1panel

**Comment by a602017206** [comment]
😂好吧，我自己研究一下，多谢了
